tests/ignite/handlers/test_ema_handler.py

```diff
@@ -0,0 +1,329 @@
+import os
+from typing import Any, Callable, Union
+
+import pytest
+import torch
+import torch.nn as nn
+from torch.nn.parallel import DataParallel, DistributedDataParallel
+from torch.utils.data import DataLoader, TensorDataset
+
+import ignite.distributed as idist
+from ignite.engine import Engine, Events
+from ignite.handlers import EMAHandler
+
+
+def _get_dummy_dataloader() -> DataLoader:
```

Comment:
Decorate these `_get_dummy_....` as `@pytest.fixture`would be nice I think, like we did in LRFinder tests here https://github.com/pytorch/ignite/blob/4d6d220e7be7a433cf2e16f92ca7cc6969ecea89/tests/ignite/contrib/handlers/test_lr_finder.py#L136

Reply:
Good idea, I will update it. Thanks.